keadm/README.md

```diff
@@ -1,17 +1,263 @@
-# KubeEdge Installer
+# Getting Started with KubeEdge Installer
 
-[Kubeedge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
+Please refer to KubeEdge Installer proposal document for details on the motivation of having KubeEdge Installer.
+It also explains the proposed functionality of the proposed commands.
+[KubeEdge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
 
-#### Build KubeEdgeinstaller
+## Limitation
 
- ```shell
-    cd $GOPATH/src/github.com/kubeedge/kubeedge/keadm
-    
-    make
-     
-    sudo mv keadm /usr/local/bin
- ```
-  
-#### Execute command to get started
+- Currently support of `KubeEdge installer` is available only for Ubuntu OS. CentOS support is in-progress.
 
-sudo keadm --help
\ No newline at end of file
+## Downloading KubeEdge Installer
+
+1. Goto [KubeEdge Release](<https://github.com/kubeedge/kubeedge/releases>) page and download `keadm-v0.3.0-beta.0-linux-amd64.tar.gz`.
+2. Untar it your desired folder, by executing `tar -xvzf keadm-v0.3.0-beta.0-linux-amd64.tar.gz`.
+3. The binary is untarred in `kubeedge` folder in current path. The binary name is `kubeedge`
+
+## Building KubeEdge Installer
+
+1. Download the source code either by
+
+- `git clone https://github.com/kubeedge/kubeedge.git $GOPATH/src/github.com/kubeedge/kubeedge`
+- Download option available at [KubeEdge git repo](<https://github.com/kubeedge/kubeedge>)
+- Download from [KubeEdge Release](<https://github.com/kubeedge/kubeedge/releases>) page
+
+2. `cd $GOPATH/src/github.com/kubeedge/kubeedge/keadm`
+3. `make`
+4. Binary `kubeedge` is available in current path
+
+## Installing KubeEdge Cloud component
+
+Referring to `KubeEdge Installer Doc`, the command to install KubeEdge cloud component (edge controller) and pre-requisites
+
+- Execute `sudo kubeedge init`
+
+**NOTE:**
+Steps  required for Device Management. For v0.3.0-beta.0, both KubeEdge installer and CRD for Device Management were release together. 
```

Comment:
We can simplify both the lines as :

Device CRD yamls need to be applied in a K8S cluster for device management. Support for deploying these CRD's as a part of the installer will be available in the next release.